script/ci/restore-staging-baseline.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Restore baseline data for staging in case a pull request messed with
+# the database.
+
+set -e
+
+: ${DB_USER='ofn_user'}
+: ${DB_DATABASE='openfoodnetwork'}
+
+srcdir="$HOME/apps/openfoodnetwork/shared"
+srcfile="$srcdir/staging-baseline.sql.gz"
+
+if [ -f "$srcfile" ]; then
+	echo "Restoring data from: $srcfile"
+else
+	>&2 echo "[Error] No baseline data available at: $srcfile"
+	exit 1
+fi
+
+dropdb -h localhost -U "$DB_USER" "$DB_DATABASE" --if-exists
```

Comment:
I think postgres blocks you from dropping a db if there's still active connections, do we need to stop puma and sidekiq?

Reply:
Note, I had an issue before when Metabase was also connected, remotely. We have a solution to force close the connection here:
https://github.com/openfoodfoundation/ofn-install/blob/master/roles/db_restore/tasks/restore_database.yml#L15-L20